### PR TITLE
Added git pill

### PR DESCRIPTION
**Pull request by Kardelio**
issue #21 

Also this is an awesome repo! Well done and Id love to contribute more!

### CODE REVIEW

File: shellGitTypo.sh [hunk 1]
```diff
@@ -95,6 +95,17 @@ gitTypoPoop="
  _(_\ \)__
 (____\___))"
 
+gitTypoPill="
```
Comment (Xaxetrov) [inline shellGitTypo.sh]: Here you do not need to escape characters. Just put the ascii art between simple quotes like this
```
'
 _________
[_________]
 |  .-.  |
 |,(o.o).|
 | >|n|< |
 |` `"` `|
 |POISON!|
 `"""""""`
'
```

Comment (Xaxetrov) [inline shellGitTypo.sh]: Please move this art just under `gitTypoExplosion7` so that they stay in alphabetical order

File: shellGitTypo.sh [hunk 2]
```diff
@@ -192,6 +203,15 @@ git() {
     curl -s http://whatthecommit.com/index.txt
     command git commit "${@:2}"
 
+  # pill
```
Comment (Xaxetrov) [inline shellGitTypo.sh]: Please replace your comments with just `# pull`

File: shellGitTypo.sh [hunk 3]
```diff
@@ -192,6 +203,15 @@ git() {
...
+  # Made a horrible yellow and purple flashing
+  # message, enjoy!
+  elif [ "$1" = "pill" ]; then
+    echo -e "\033[93;105;5m---------------------------------\033[0m"
```
Comment (Xaxetrov) [inline shellGitTypo.sh]: Nice one but we have nothing of color yet and I am not a fan of messing with user's colors. Could you just print the text without it ? Or was it part of the joke and I did not get it ?

File: shellGitTypo.sh [hunk 4]
```diff
@@ -192,6 +203,15 @@ git() {
...
+    echo -e "\033[93;105;5mNow thats a hard pill to swallow!\033[0m"
+    echo -e "\033[93;105;5m---------------------------------\033[0m"
+	echo "$gitTypoPill"
```
Comment (Xaxetrov) [inline shellGitTypo.sh]: This line has a tab instead of 4 spaces. Please check that all your indentation is made of spaces.

**Review by Xaxetrov — COMMENTED** [review]
Thanks for your contribution!

File: shellGitTypo.sh (hunk 1 above)
Comment (Kardelio) [reply shellGitTypo.sh]: Oooooo didnt know that! thats awesome thanks for that! #alwaysLearning!

File: shellGitTypo.sh (hunk 3 above)
Comment (Kardelio) [reply shellGitTypo.sh]: Understood! wil remove color and nah it was just me being excited and making it look crazy haha will remove it now

**Comment by Kardelio** [comment]
@Xaxetrov updated the PR with a new commit with all your suggestions.
How is it now?
Thanks for your feedback btw!

File: shellGitTypo.sh [hunk 5]
```diff
@@ -192,6 +203,11 @@ git() {
     curl -s http://whatthecommit.com/index.txt
     command git commit "${@:2}"
 
+  # pull
```
Comment (Xaxetrov) [inline shellGitTypo.sh]: Please move the section just above the "# remote" section so that sections stay ordered alphabetically.

File: shellGitTypo.sh [hunk 6]
```diff
@@ -192,6 +203,11 @@ git() {
...
+  elif [ "$1" = "pill" ]; then
+    echo "Now thats a hard pill to swallow!"
+    echo "$gitTypoExplosion7"
+
```
Comment (Xaxetrov) [inline shellGitTypo.sh]: Do not forget to call the actual command, with arguments.

File: shellGitTypo.sh (hunk 5 above)
Comment (Kardelio) [reply shellGitTypo.sh]: Will do hold on :D

File: shellGitTypo.sh (hunk 6 above)
Comment (Kardelio) [reply shellGitTypo.sh]: Yeh totally forgot this and jsut saw it myself, hold on...

**Comment by Kardelio** [comment]
@Xaxetrov I really appreciate you giving lots of great feedback on the PR, sorry i missed a bunch of things, but just submitted two more commits and hopefully they are ok now?
Thanks again I love the attention to detail and honest feedback! Very cool! Again please let me know if there are any more things that need dealing with

Also is it possible to squash my commits into a single one, if and when, the PR gets accepted?

File: shellGitTypo.sh [hunk 7]
```diff
@@ -87,6 +87,16 @@ gitTypoExplosion7='
      ) /
     ,---.'
     
+gitTypoExplosion8='
```
Comment (Xaxetrov) [inline shellGitTypo.sh]: Why did you change the name ?
gitTypoPill was OK.

I was maybe not very clear in a previous comment, but I was just complaining about keeping constant names in order.

**Review by Xaxetrov — COMMENTED** [review]
Just one small pb left.

I will squash the commits when merging.
You are welcome to contribute more ! You can:
- Pick other open issues
- Create new ones if you have ideas
- Directly do other PRs if you have ideas and want to do them yourself right away.

File: shellGitTypo.sh (hunk 7 above)
Comment (Kardelio) [reply shellGitTypo.sh]: Ahhhh sorry yeah i got a little confused with that, i have now changed the name back to how it was, sorry for my confusion, my bad
